src/data/complex/basic.lean

```diff
@@ -468,6 +479,9 @@ calc abs 2 = abs (2 : ℝ) : by rw [of_real_bit0, of_real_one]
 lemma abs_nonneg (z : ℂ) : 0 ≤ abs z :=
 real.sqrt_nonneg _
 
+@[simp] lemma range_abs : range abs = Ici 0 :=
+subset.antisymm (range_subset_iff.2 abs_nonneg) $ λ x hx, ⟨x, abs_of_nonneg hx⟩
```

Comment:
Is it worth adding this for `norm` and `nnorm` too?

Reply:
I'll do it for any nontrivial real normed space in another PR.